tooltester/src/main/java/io/dockstore/tooltester/TooltesterConfig.java

```diff
@@ -14,38 +14,49 @@
  */
 public class TooltesterConfig {
 
-    private HierarchicalINIConfiguration config;
+    private HierarchicalINIConfiguration tooltesterConfig;
+    private HierarchicalINIConfiguration dockstoreConfig;
     private String jenkinsServerUrl;
     private String serverUrl;
     private String[] runner;
     private String dockstoreVersion;
     private String s3Bucket;
     private String s3Endpoint;
+    private String dockstoreAuthorizationToken;
 
     public TooltesterConfig() {
         String userHome = System.getProperty("user.home");
         try {
             // This is the config file for production use.
             // If there's no configuration file, all properties will default to Travis ones.
             File configFile = new File(userHome + File.separator + ".tooltester" + File.separator + "config");
-            setConfig(new HierarchicalINIConfiguration(configFile));
-            setServerUrl(config.getString("server-url", "https://staging.dockstore.org/api"));
-            setRunner(config.getString("runner", "cwltool cwl-runner cromwell").split(" "));
-            setDockstoreVersion(config.getString("dockstore-version", "1.6.0"));
-            setS3Bucket(config.getString("s3-bucket", "dockstore.tooltester.backup"));
-            setS3Endpoint(config.getString("s3-endpoint", "https://s3.amazonaws.com"));
-            setJenkinsServerUrl(config.getString("jenkins-server-url", "http://172.18.0.22:8080"));
+            setTooltesterConfig(new HierarchicalINIConfiguration(configFile));
+            setRunner(tooltesterConfig.getString("runner", "cwltool cwl-runner cromwell").split(" "));
+            setDockstoreVersion(tooltesterConfig.getString("dockstore-version", "1.13.1"));
```

Comment:
My previous comment in the code you moved here wasn't clear; I was referring to the hard-coded `1.13.1` version. That means if we change the Dockstore version we're running, we have to remember to update this, or they'll be out of sync.

I'm not clear if this is the CLI version or the web service version -- perhaps the the field name could describe that, e.g., 'dockstoreCliVersion` (or maybe capture both?).

In any either or both cases, my suggestion is to get the version dynamically:

* If it's the CLI version, run `dockstore -version` and capture the version
* If it's the web service version, get the web service version via an API request.

Reply:
For this tool, the CLI version has zero impact. That was from previous code in tooltester. My code relies on whatever version of the dockstore CLI the user has installed.

Reply:
Do you think I should add a comment explaining this to the **TooltesterConfig.java**.

Reply:
I see, checking... hmm, looks like it isn't used at all, by anything. :) I only commented on it because I saw the code had changed.

If you can be sure it's not used, I would just delete it; if you're unsure, yes, optionally add a comment so the next time somebody updates the code, this discussion doesn't happen again :).

Reply:
Comment added. It looks like it might be used [here](https://github.com/dockstore/dockstore-support/blob/d3339484b71ad701e4e1b2eb943b902ba5065fba/tooltester/src/main/java/io/dockstore/tooltester/client/cli/Client.java#L335).

Reply:
D'oh, I did a case sensitive search. Should have loaded the project in the IDE.